sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLDialectSpecTest.java

```diff
@@ -658,7 +658,6 @@ public void testIfNullNegative() {
   }
 
   @Test
-  @Ignore("Throws IndexOutOfBoundsException")
   public void testConstructEmptyArrayLiteral() {
```

Comment:
Since a similar new test is added, can we rename this one to `testEmptyArrayParameter`? That makes more sense to me.

Reply:
Done.